src/components/page/reference/ReleaseNotes/index.tsx

```diff
@@ -6,17 +6,21 @@ import styles from './styles.module.css';
 
 interface Release {
   body: string;
-  element: string;
   name: string;
   published_at: string;
-  symbol: string;
   tag_name: string;
   type: string;
   version: string;
 }
 
 export default function ReleaseNotes(props: { [key: string]: any }) {
   if (releases.length === 0) {
+    console.warn('Could not load release notes data. Make sure that you have a valid GITHUB_TOKEN. ');
+    console.warn('Create a personal access token by following the below guide:');
+    console.warn('https://docs.github.com/en/enterprise-cloud@latest/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token');
+    console.warn('and then authorize it to work with SSO:');
+    console.warn('https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on');
```

Comment:
Nitpick: Could we combine these into one warning? If you use \`template strings\`, you can include line breaks.

Reply:
Yeah, absolutely! Done here: https://github.com/ionic-team/ionic-docs/pull/2921/commits/5f195f6b3951ae9044f563d09e78bca2f3240e78